PollApp.ts

```diff
@@ -177,7 +177,20 @@ export class PollApp extends App implements IUIKitInteractionHandler {
             }
 
             case 'mode': {
-                const modal = await createPollModal({ id: data.container.id, data, persistence, modify, mode: data.value });
+                const viewId = data.container.id;
+                const viewAssociation = new RocketChatAssociationRecord(RocketChatAssociationModel.MISC, viewId);
+                const optionsAssociation = new RocketChatAssociationRecord(RocketChatAssociationModel.MISC, 'options');
+
+                const existingOptions: undefined | { options: number } = await read.getPersistenceReader()
```

Comment:
This gives `existingOptions` as `undefined` which doesn't solve the problem. Enforcing type as `any` solves it. Not sure how.


https://user-images.githubusercontent.com/28918901/133000932-b9e54d8c-6647-49be-8b12-94f7f20b36b9.mp4

Reply:
Hmm... that's definitely weird :thinking: